frontend/src/libraries/explorer-wamp/details.ts

```diff
@@ -0,0 +1,67 @@
+import { ExplorerApi } from ".";
+
+export interface Details {
+  accountsCount: number;
+  totalNodesCount: number;
+  onlineNodesCount: number;
+  lastDayTxCount: number;
+  lastBlockHeight: number;
+  transactionsPer10Second: number;
+  transactionsPerSecond: number | null;
+}
+
+export default class DetailsApi extends ExplorerApi {
+  async getDetails(): Promise<Details> {
+    try {
+      const blockTimestamp = await this.call("select", [
+        `SELECT timestamp FROM blocks WHERE timestamp > ((strftime('%s','now') - 30) * 1000 ) ORDER BY timestamp DESC LIMIT 1`
```

Comment:
Translate your code into simple English and find that you end up taking into account more transactions than you want.

1. Let's define NOW to be the current timestamp.
2. You query the block that is not older 30 seconds from now, and is the latest (great! but...)
3. ... you wait here for the response, let's say a minute, just to see how ridiculous the result will be
4. You make a new query with:

    ```
    SELECT COUNT(*) as transactionsPer10Second FROM transactions
              WHERE block_timestamp > (:blockTimestamp - 10 * 1000)  ) as transactions
    ```

    You count the number of transaction happened from the latest block a while ago (since that time we might have already synced new blocks) to the NEW NOW moment.

Add `AND block_timestamp <= :blockTimestamp`.

Please, don't make our database doing extra work when it can avoid doing it -- don't make it count with `strftime('%s','now') * 1000` if you are not going to (and should not) use it anyway.